src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q11/Bonus.java

```diff
@@ -0,0 +1,35 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q11;
+/*
+Your professor has decided to give everyone in the class a bonus 10% on an exam! Write a java program that asks the
```

Comment:
Change question text to:
Your professor has decided to give everyone in the class a bonus of 10% on an exam! Write a java program asking the user how many marks the exam is out of (as an integer) and the grade they received (as a double). Then, calculate their new score with the bonus, and print this new score with a formatted string to round the percentage to two decimal places.

Reply:
Also, make sure to update the document when you enter this question